src/main/java/org/eclipse/golo/runtime/ClassReferenceSupport.java

```diff
@@ -55,7 +55,17 @@ public static CallSite bootstrap(MethodHandles.Lookup caller, String name, Metho
     try {
       return Class.forName(name, true, classLoader);
     } catch (ClassNotFoundException e) {
-      return null;
+      switch(name) {
```

Comment:
@yloiseau use `Map<String, Class>` cache seems to me not a good idea according `Class` instances are classloader dependent and some project use multiple classloaders (even if there really is no case primitive types to be in two different classloaders)
@jponge yes, I had in mind to extract a method and check first if it is a primitive type (not in the catch clause)